conf/citest/pipelines/DumpGenomes.json

```diff
@@ -0,0 +1,39 @@
+{
+    "reference_db": "mysql://ensro@mysql-ens-compara-prod-8:4618/jalvarez_citest_dump_genomes_99",
+    "reference_dir": "/hps/nobackup2/production/ensembl/jalvarez/genome_dumps",
```

Comment:
As a convention, once the reference datasets are complete, should we:
- move the files to the shared user's space `/nfs/production/panda/ensembl/warehouse/compara/citest/`? (it has to be on a space that is backed up)
- remove the username from the database name, e.g. → `citest_dump_genomes_99`

Reply:
I would propose that all of the CITest dbs should live on a single server too (`mysql-ens-compara-prod-10`, maybe?)